practice-areas/engineering/accessibility.md

```diff
@@ -4,22 +4,22 @@ title: Accessibility
 
 # Accessibility
 
-We implement Section 508 standards and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+We implement Section 508 compliant sites and strive to meet the latest Web Content Accessibility Guidelines (WCAG). We do this so that everyone can access our sites, even if they have a permanent, temporary or situtational disability. The Engineering team is very aware of the advantages of open standards, and semantic code for building future-compatible applications. CivicActions has an [Accessibility Practice Area](/en/latest/practice-areas/accessibility/), but is also a component of all other practice areas at CivicActions.
 
 ## How we do this
 
--   We use base themes which strive to meet the W3C's WCAG 2.0 AA Success Criteria or above. This will help satisfy the requirements of the ADA.
+-   We use base themes which strive to meet the W3C's WCAG 2.1 AA Success Criteria or above.
 -   We implement good SEO structures (which are generally good for accessibility).
 -   We implement responsive design strategies that can present complex data in hierarchical structures, capable of being navigated by audio cues.
 -   We believe that users should be able to personalize how content is presented. We strive to allow users to change color schemes and text size displays allow them to have a better user experience.
--   We test sites using a combination of automated and manual testing.
--   We use good practices in using semantic markup in page templates and report design to define content hierarchy (e.g. primary head is h1, secondary head is h2, etc.).
+-   We test sites using a combination of automated and manual testing. We leverage automated testing in the browser, with site-wide scans and also through integration in our CI/CD pipeline.
+-   We follow the best practices of the Drupal community, which includes accessibility.
 
 ## When we do this
 
 -   We strive to produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
 -   We aim to do accessibility work continuously, as part of our agile process. Accessibility scans should be performed on a per-ticket basis and signed off on before work is considered complete.
--   Accessibility should NOT BE left until the end of a project, but baked in throughout its lifespan.
+-   We know that accessibility cannot NOT BE left until the end of a project, but baked in throughout its lifespan.
```

Comment:
We know that accessibility CANNOT be left

Reply:
Ok. For all of you who don't enjoy double negatives and inconsistent case. :)  
  
Thanks @jastraat